backend/open_webui/tests/test_outlook_mcp_provider.py

```
@@ -0,0 +1,114 @@
+        provider.search_emails(
+            "user@example.com", "subject:Project X OR body:Project X", max_results=5
+        )
+    )
+
+    assert result["content"].startswith("Found 0 email(s)")
+
+
+def test_search_contacts_escapes_nested_quotes(monkeypatch):
+    provider = OutlookMCPProvider(
+        client_id="client",
+        client_secret="secret",
```

Ruff (0.13.3)

Possible hardcoded password assigned to argument: "client_secret"

(S106)